### ISSUE TITLE
Would you consider to create a 'Liquid' SFML2 Debug Draw? 

### ISSUE THREAD

**Issue by DJuego** [MrPlow442/Box2D-SFML2-Debug-Draw#1]
I am using your Debug Draw with Liquid. It works perfectly! Thank you very much!
(http://google.github.io/liquidfun/)

However, i miss the DrawParticles method...

void SFMLDebugDraw::DrawParticles(const b2Vec2 *centers, float32 radius, const b2ParticleColor *colors, int32 count)

I am not sure that it be possible...  I do not know OpenGL. :-(.
In any case, thanks again!

DJuego

**Comment by MrPlow442** [comment]
I haven't worked on SFML debug draw in quite a while. Unfortunately I am currently quite busy with work so, at this moment, I cannot devote my time to implement this feature. From what I've seen it shouldn't be too difficult to implement as the testbed particles are just a bunch of triangles, though the [implementation](https://github.com/google/liquidfun/blob/0708ce1464b0db3cf34353258ce939a82f28a173/liquidfun/Box2D/Testbed/Framework/Render.cpp#L129) looks a bit more complex. 

As I said, I am currently too busy with work to devote my time to this, however I might implement this in a few months when I find the time, or if you wish you can try to implement it yourself and hit me up with a PR.

**Comment by DJuego** [comment]
Right. It is fair and it is enough! :-}.  Thank you for your swift answer!
